core/src/index.ts

```diff
@@ -1056,7 +1066,8 @@ class Store implements Subscribable {
     set(path: Path, value: StateValueAtPath): SetActionDescriptor {
         if (this.edition < 0) {
             // TODO convert to console log
```

Comment:
maybe remove TODO comment? and update web docs for this error code?

Reply:
I removed this block entirely, as the warnings were excessive in our usage, and there are legitimate reasons for updating global state while a component unmounts (updating hooked global state on unmount).